opennav_row_coverage/test/test_server.cpp

```diff
@@ -132,7 +132,7 @@ TEST(ServerTest, testDynamicParams)
     node->get_node_services_interface());
 
   auto results = rec_param->set_parameters_atomically(
-    {rclcpp::Parameter("default_turn_point_distance", 0.25),
+    {rclcpp::Parameter("default_turn_point_distance", 0.25),  // Discretization ??
```

Comment:
?

Reply:
The turn_point_distance used to be a member variable of path_generator.  The API change appears to have changed this to a member function SetDiscretization on the class f2c::pp::TurningBase.

I can remove the comment.  The comment was a reminder to check if my interpretation of the api change was correct and whether the parameter name should be changed to better reflect what it is used for within Fields2Cover.  

https://github.com/Fields2Cover/Fields2Cover/blob/4b57a70dcac7f10cdd4760edbacdb0b48fdbc1c8/include/fields2cover/path_planning/turning_base.h#L69

Reply:
removed comment.